public/components/tweets/tweets.view.html

```diff
@@ -0,0 +1,38 @@
+<nav id="nav">
+    <div class="nav-wrapper">
+        <a class="brand-logo left">
+            Twitter Analyzer
+        </a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+            <li><a ng-click="scrollTo('intro')">About</a></li>
+            <li><a ng-click="scrollTo('analyze')">Analyze</a></li>
+        </ul>
+    </div>
+</nav>
+<div class="heading">
+	<div class="container">
+		<div class="row">
+			<div class="col s12 m10 offset-m1">
```

Comment:
Spacing should be consistent, either 2 or 4 spaces for your indentation.  You and Asa should agree on which amount.  The nav above uses 4 spaces, but this block here all appears to be using 2 tabs which causes some display differences when looking at the code.  My suggestion for HTML and JavaScript markup is to strictly use 2 spaces, not tabs.